src/subcommands/arc_diff_against_parent.py

```diff
@@ -28,6 +28,10 @@ def run_command(self, args):
         # type: (Namespace) -> None
         extra_arc_diff_options = args.arc_diff_args
         if extra_arc_diff_options:
+            # If the first extra arg starts with "-", "--" must also have been passed, and
+            # argparse doesn't remove it for us
+            if "--" in extra_arc_diff_options:
```

Comment:
`.remove()` only removes the first matching entry already, but I'll also change it to just check the zeroth item.